build/benchmarks.matrix.01.yml

```diff
@@ -219,6 +219,13 @@ groups:
       - amd-lin-load
       - amd-lin2-db # required by nodejs and tfb-database argument
 
+    - name: Trends Siryn Linux
+      template: trend-scenarios.yml
+      profiles:
+      - siryn-lin-app
```

Comment:
Only the database version needs a db agent

Reply:
Good catch, fixed 👍